packages/loader/container-definitions/src/runtime.ts

```diff
@@ -80,6 +80,8 @@ export interface IRuntime extends IDisposable {
      */
     stop(): Promise<IRuntimeState>;
 
+    raceToSummarize(): Promise<void>;
```

Comment:
comments? also i don't love adding a new api for this. need to think if there is a way around it

Reply:
Removed new API, added raceToSummarize as default behavior within `stop()` in a future PR we should probably add a new API and deprecate stop.